public/components/model_version/model_version.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useState, useEffect, useCallback } from 'react';
+import {
+  EuiButton,
+  EuiPageHeader,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingSpinner,
+} from '@elastic/eui';
+import { generatePath, useHistory, useParams } from 'react-router-dom';
+
+import { useFetcher } from '../../hooks';
+import { APIProvider } from '../../apis/api_provider';
+import { routerPaths } from '../../../common/router_paths';
+import { VersionToggler } from './version_toggler';
+
+export const ModelVersion = () => {
+  const { id: modelId } = useParams<{ id: string }>();
+  const { data: model } = useFetcher(APIProvider.getAPI('model').getOne, modelId);
+  const [modelInfo, setModelInfo] = useState<{ version: string; name: string }>();
+  const history = useHistory();
+  const modelName = model?.name;
+  const modelVersion = model?.model_version;
+
+  const onVersionChange = useCallback(
```

Comment:
Shall we simply make version a link? Like:
```typescript
<Link to={generatePath(routerPaths.modelVersion, { id })}>1.0.1</Link>
```

Reply:
I need to set the state first, or the old version number will be displayed until the new model version data loaded. Since the switch version list design isn't finalized, I think we can keep it simple first.

Reply:
> I need to set the state first, or the old version number will be displayed until the new model version data loaded.
We can display a loading indicator when it's loading

> Since the switch version list design isn't finalized, I think we can keep it simple first.
Sure, let's do this way and get back to this later

Reply:
The main purpose of current implementation is to avoid the page header disappear after version change. It will be a little weird if it's disappear after select new version. It's more simple to show a page loading indicator after version change.

Reply:
I see the point, that makes sense to me